content/en/cd-as-a-service/concepts/deployment/how-hpa-works.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "How CD-as-a-Service implements Kubernetes HPA"
+linktitle: "HPA"
+description: >
+  Armory CD-as-a-Service supports Kubernetes deployments with HPA.
```

Comment:
I made some changes that do not allow me to commit this change - I will make it as my own commit because i like it and struggled with how to SD this topic.